Improve documentation references in `README` (#1124)
entropyxyz/entropy-core: README.md

## Documentation

- High level introduction to Entropy: [docs.entropy.xyz](https://docs.entropy.xyz)

> - API documentation for this crate: [docs.rs/entroyp-ts](https://docs.rs/entropy-tss/latest/entropy_tss/index.html)

Reviewer: ```suggestion
- API documentation for the `entropy-tss` crate: [docs.rs/entropy-tss](https://docs.rs/entropy-tss/latest/entropy_tss/index.html)
```

You can also build the API docs yourself:
1. [Install the dependencies](#building-from-source)
2. Invoke
    ```bash
    cargo doc --no-deps --open`
    ```

> There is also [high level documentation for Entropy available here](https://docs.entropy.xyz).

Reviewer: We can get rid of this line now

Author: ```suggestion
```
